tests/commands/controlled_lists_import_export_tests.py

```diff
@@ -0,0 +1,29 @@
+from django.conf import settings
+from django.core import management
+from arches.app.models.models import (
+    ControlledList,
+    ControlledListItem,
+    ControlledListItemImage,
+    ControlledListItemImageMetadata,
+    ControlledListItemValue,
+    DValueType,
+    Language,
+)
+
+from tests.base_test import ArchesTestCase
+
+
+# these tests can be run from the command line via
+# python manage.py test tests.commands.controlled_lists_import_export_tests --settings="tests.test_settings"
+
+
+class ControlledListsImportExportTests(ArchesTestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        from tests.views.controlled_lists_tests import ControlledListTests
+        return ControlledListTests.setUpTestData()
+
+    
+    def test_export_controlled_list(self):
+        management.call_command("packages", operation='export_controlled_lists', dest_dir=settings.TEST_ROOT)
```

Comment:
To clean up this file after the test runs, you can do this before the call_command:
```py
        dest = Path(TEST_ROOT) / "path_to_the_sheet.xlsx"  # needs `from pathlib import Path` at top of file
        self.addCleanup(os.remove, dest)
```

That way, whether call_command passes or fails, the os.remove() will be attempted and will delete your sheet. It's a lot like wrapping the meat of your test in a try/finally. The callback that you would ordinarily put in the finally is what you pass to addCleanup.

Tangent: If call_command fails so early that it doesn't even write a sheet, your cleanup might raise FileNotFoundError, but your test is failing anyway in that scenario, so two errors in that scenario isn't a developer experience terrible enough to spend more time making the cleanup function more robust.